zookeeper-server/src/main/java/org/apache/zookeeper/common/X509Util.java

```diff
@@ -90,6 +89,7 @@
     private String sslTruststoreLocationProperty = getConfigPrefix() + "trustStore.location";
     private String sslTruststorePasswdProperty = getConfigPrefix() + "trustStore.password";
     private String sslTruststoreTypeProperty = getConfigPrefix() + "trustStore.type";
+    private String sslClientContextProperty = getConfigPrefix() + "client.context";
```

Comment:
nits:
- not sure why "client" is in the name
- let's call the property something like "context.supplier.class"?